ichnaea/locate/tests/test_searcher.py

```diff
@@ -12,7 +12,7 @@
 )
 
 
-class SearcherTest(DBTestCase, GeoIPIsolation):
+class SearcherTest(GeoIPIsolation, DBTestCase):
```

Comment:
Any specific reason to reorder these? I think I stuck to putting the testcase subclass as the first one in all other places. The *Isolation classes never implement any of the setUp/tearDown/class methods, but I kept the testcase as the first one to make sure none of them sneak in and make the super calls even more complicated.